app/controllers/hub/automated_messages_controller.rb

```diff
@@ -10,46 +10,23 @@ def index
     private
 
     def messages_preview
-      automated_messages = [
-        [AutomatedMessage::SuccessfulSubmissionDropOff, {}],
-        [AutomatedMessage::SuccessfulSubmissionOnlineIntake, {}],
-        [SurveyMessages::GyrCompletionSurvey, { survey_link: "https://fakecodeforamerica.co1.qualtrics.com" }],
-        [SurveyMessages::CtcExperienceSurvey, { survey_link: "https://fakecodeforamerica.co1.qualtrics.com" }],
-        [AutomatedMessage::DocumentsReminderLink, { body_args: { doc_type: "ID" } }],
-        [AutomatedMessage::EfileAcceptance, {}],
-        [AutomatedMessage::EfilePreparing, {}],
-        [AutomatedMessage::EfileRejected, {}],
-        [AutomatedMessage::EfileRejectedAndCancelled, {}],
-        [AutomatedMessage::EfileFailed, {}],
-        [AutomatedMessage::CtcGettingStarted, {}],
-        [AutomatedMessage::ClosingSoon, {}],
-        [AutomatedMessage::SaveCtcLetter, { body_args: { service_name: MultiTenantService.new(:ctc).service_name } }],
-        [AutomatedMessage::ContactInfoChange, {}],
-        [AutomatedMessage::FirstNotReadyReminder, {}],
-        [AutomatedMessage::SecondNotReadyReminder, {}],
-        [AutomatedMessage::InformOfFraudHold, {}],
-        [AutomatedMessage::NewPhotosRequested, {}],
-        [AutomatedMessage::VerificationAttemptDenied, {}],
-        [AutomatedMessage::Ctc2022OpenMessage, {}],
-        [AutomatedMessage::PuertoRicoOpenMessage, {}],
-        [AutomatedMessage::IntercomForwarding, {}],
-        [AutomatedMessage::UnmonitoredReplies, { support_email: Rails.configuration.email_from[:support][:gyr] }],
-        [AutomatedMessage::InProgress, {}],
-        [AutomatedMessage::Welcome, {}],
-      ]
+      Rails.application.eager_load!
```

Comment:
Will this make it likely that there is a second or so delay when this is run for the first time?

Reply:
`config.eager_load` is set to true for all environments except for `development` in `config/environments/shared_deployment_config.rb` (and is explicitly set to false in `development.rb`), so it shouldn't - but nonetheless, this is probably not a pattern we want to use on user-facing pages.